breakglass.go

```diff
@@ -60,15 +69,7 @@ func loadAuthorizedKeys(path string) (map[string]bool, error) {
 		if tr := strings.TrimSpace(s.Text()); tr == "" || strings.HasPrefix(tr, "#") {
 			continue
 		}
-		pubKey, comment, _, _, err := ssh.ParseAuthorizedKey(s.Bytes())
-
-		// This warning can be removed once the mentioned issue is resolved
-		if keyType := pubKey.Type(); keyType == "ssh-rsa" {
-			log.Print("Warning: You added a ssh-rsa key to your authorized keys, these do currently not work.")
-			log.Print("Further information: https://github.com/gokrazy/breakglass/issues/11")
-			log.Printf("Affected key: %s [...] %s (line %d)", keyType, comment, lineNum)
-		}
-
+		pubKey, _, _, _, err := ssh.ParseAuthorizedKey(s.Bytes())
```

Comment:
Can you rebase your PR? I already removed this with https://github.com/gokrazy/breakglass/commit/44b3fe64f19c5381cb7c654c45352a171b84a0ca

Reply:
Done.